src/app/projects/components/project-settings/components/embeddings/embeddings.component.html

```diff
@@ -0,0 +1,245 @@
+                <button
+                    class="ml-1 inline-block items-center px-2.5 py-1.5 border border-gray-300 shadow-sm text-xs font-semibold rounded-md text-gray-700 bg-white hover:bg-gray-50 focus:outline-none cursor-pointer"
+                    [routerLink]="['/model-download']" [disabled]="!isManaged"
+                    [ngClass]="!isManaged ? 'cursor-not-allowed' : 'opacity-100 cursor-pointer'">
+                    <svg xmlns="http://www.w3.org/2000/svg"
+                        class="inline-block icon icon-tabler icon-tabler-arrow-autofit-down" width="20" height="20"
+                        viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round"
+                        stroke-linejoin="round">
+                        <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+                        <path d="M12 20h-6a2 2 0 0 1 -2 -2v-12a2 2 0 0 1 2 -2h8"></path>
+                        <path d="M18 4v17"></path>
+                        <path d="M15 18l3 3l3 -3"></path>
+                    </svg>
+                    <span class="leading-5">
+                        See downloaded models
+                    </span>
+                </button>
+            </div>
+
+        </div>
+    </div>
+</div>
+
+<kern-modal [isOpen]="settingModals.embedding.delete.open" [modalBoxStyle]="{'width':'30rem','text-align':'center'}"
+    [abortButton]="{
+    buttonCaption: 'Delete embedding',
+    emitObject: this, emitFunction: deleteEmbedding
+}" closeButton="X" (optionClicked)="settingModals.embedding.delete.open = false">
+    <div class="flex flex-grow justify-center text-lg leading-6 text-gray-900 font-medium">
+        Warning </div>
+    <p class="mt-2 text-gray-500 text-sm">Are you sure you want to
+        delete
+        this embedding?</p>
+    <p class="text-gray-500 text-sm">This will delete all
+        corresponding
+        tensors!</p>
+</kern-modal>
+
+
+<kern-modal [isOpen]="settingModals.embedding.create.open" [modalBoxStyle]="{'overflow-x':'visible'}" closeButton="X"
+    [acceptButton]="{
+    buttonCaption: 'Add embedding',
+    disabled: settingModals.embedding.create.blocked || settingModals.embedding.create.embeddingCreationFormGroup?.get('embeddingHandle').value == '',
+    emitObject: this, emitFunction:addEmbedding
+}" (optionClicked)="settingModals.embedding.create.open = false">
+    <div class="flex flex-grow justify-center text-lg leading-6 text-gray-900 font-medium">
+        Add an embedding </div>
+    <div class="mb-2 flex flex-grow justify-center text-sm text-gray-500">
+        Pick from the below solutions to build a vector representation</div>
+    <div *ngIf="attributesArrayTextUsableUploaded && settingModals.embedding.create.embeddingCreationFormGroup">
+        <form [formGroup]="settingModals.embedding.create.embeddingCreationFormGroup"
```

Comment:
didn't we decide against formgroups? unsure though :D

Reply:
Resolving the conversation -> removing form groups should be done in the next refactoring steps